### ISSUE TITLE
Clarity not working on individual SharePoint site

### ISSUE THREAD

**Issue by gprowlands** [microsoft/clarity#409]
No data is being logged in Clarity from SharePoint sites. Error 400 bad request shown in the network trace. Anyone else having trouble with SharePoint sites/clarity?

**Comment by ozzymuppet** [comment]
Yes, same here. Microsoft have even adding a warning to their instructions.

" Important
It was found on May 15, 2023, that certain customers are having trouble with single-site installations. As of the latest update on June 08, 2023, we are actively working to resolve these."

June 2023. It's now August 2023, still not working. That's a concern!

**Comment by LWCCUCode** [comment]
It's November 2024 and it seems this is still an open issue? I just added Clarity to a single SharePoint site and I'm getting nothing in the Dashboard even though I don't see any issues when looking at the Network trace, I see Clarity is loading without error but nothing is being reported in the Dashboard.

**Comment by ClaritySupport** [comment]
As per the recent communication with SharePoint contacts. Single site installation is no longer be supported in Clarity integration with SharePoint for the time being. The documentation will be updated to indicate that.
